docs/02-app/02-api-reference/08-next-cli.mdx

```diff
@@ -18,59 +18,79 @@ _([npx](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7
 The output should look like this:
 
 ```bash filename="Terminal"
-Usage
-  $ next <command>
-
-Available commands
-  build, start, export, dev, lint, telemetry, info
-
-Options
-  --version, -v   Version number
-  --help, -h      Displays this message
-
-For more information run a command with the --help flag
-  $ next build --help
+Usage next [options] [command]
+
+The Next.js CLI allows you to develop, build, and start your application, and more.
+
+Options:
+  -v, --version                Outputs the Next.js version.
+  -h, --help                   Displays this message.
+
+Commands:
+  build [directory] [options]  Creates an optimized production build of your application.
+                               The output displays information about each route.
+  dev [directory] [options]    Starts Next.js in Development mode with hot-code reloading,
+                               error reporting, and more.
+  info [options]               Prints relevant details about the current system which can be
+                               used to report Next.js bugs.
+  lint [directory] [options]   Runs ESLint for all files in the `/src`, `/app`, `/pages`,
+                               `/components`, and `/lib` directories. It also provides a
+                               guided setup to install any required dependencies if ESLint
+                               is not already configured in your application.
+  start [directory] [options]  Starts Next.js in production mode. The application should be
```

Comment:
```suggestion
  start [directory] [options]  Starts Next.js in Production mode. The application should be
```

Reply:
Looks like we're lowercasing further down. Example → https://github.com/vercel/next.js/pull/62304/files#diff-b1f898409774308431726a8d0e159f3f5562cc31d112721f8dbb2f963ae86629R58

Reply:
@manovotny Good catch! I need to make sure the actual help output is lowercased.